src/Properties.ts

```diff
@@ -5,11 +5,32 @@ import * as fs from 'fs'
 
 const log = Logger.getLogger('applicationRunner')
 
-export function addTo (config: any, mountPoint: fs.PathLike = '/mnt/secrets/') {
+interface Options {
+  mountPoint?: fs.PathLike
+  failOnError?: boolean
+}
+
+const defaultOptions: Options = {
+  mountPoint: '/mnt/secrets/',
+  failOnError: false
+}
+
+export function addTo (config: any, givenOptions?: Options) {
+  const options: Options = merge({}, defaultOptions, givenOptions || {})
+  const mountPoint: fs.PathLike = options.mountPoint!
+  const failOnError: boolean = options.failOnError!
+
   log.info(`Reading properties from volume: '${mountPoint}'`)
-  const prefix = getPrefix(mountPoint.toString())
-  const properties = readVaults(mountPoint)
-  config[prefix] = merge(config[prefix] || {}, properties)
+  try {
+    const prefix = getPrefix(mountPoint.toString())
+    const properties = readVaults(mountPoint)
+    config[prefix] = merge(config[prefix] || {}, properties)
+  } catch (error) {
+    if (failOnError) {
+      throw Error(`properties-volume failed with:'${error}`)
+    }
+    log.info(`Reading properties from volume: '${mountPoint}' FAILED with '${error}'`)
```

Comment:
debug? it's probably normal operation to not be there during the transition period

Reply:
well its still info not debug because its information about the failure rather than to trace a failure.
Normally production has only `> warn` enabled.